src/main/java/Models/Prescription/Prescription.java

```diff
@@ -0,0 +1,49 @@
+package Models.Prescription;
+
+/**
+ *
+ * @author Bredan
+ */
+
+public class Prescription {
+    private final Integer patientID;
+    private final String medicine;
+    private final Integer quantity;
+    private final Boolean repeating;
+    private final String issueDate;
+    private final String endDate;
```

Comment:
Same for this type

Reply:
this type is also converted in the prescriptionDAO. Although not explicitly converted to date in the prescription object.

Reply:
changed to reflect date type being in the prescription object.